<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditSnippets</class>
 <widget class="QDialog" name="EditSnippets">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Snippets</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QListWidget" name="ui_lwSnippetFiles">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="ui_pbAddFile">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ui_pbDeleteFile">
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Label:</string>
       </property>
       <property name="buddy">
        <cstring>ui_leBookLabel</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="ui_leBookLabel"/>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QTableWidget" name="ui_twSnippets">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <column>
        <property name="text">
         <string>Label</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Icon</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Tex</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Pos</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="ui_pbAddSnippet">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ui_pbDeleteSnippet">
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QFrame" name="ui_frSnippet">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Label</string>
        </property>
        <property name="buddy">
         <cstring>ui_leSnippetLabel</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="ui_leSnippetLabel"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Icon</string>
        </property>
        <property name="buddy">
         <cstring>ui_leSnippetIcon</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLineEdit" name="ui_leSnippetIcon"/>
        </item>
        <item>
         <widget class="QToolButton" name="ui_tbSnippetIcon">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Tex</string>
        </property>
        <property name="buddy">
         <cstring>ui_pteSnippetTex</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPlainTextEdit" name="ui_pteSnippetTex">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Pos</string>
        </property>
        <property name="buddy">
         <cstring>ui_spbSnippetPos</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="ui_spbSnippetPos">
        <property name="minimum">
         <number>-1</number>
        </property>
        <property name="maximum">
         <number>9999</number>
        </property>
        <property name="value">
         <number>-1</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ui_lwSnippetFiles</tabstop>
  <tabstop>ui_pbAddFile</tabstop>
  <tabstop>ui_pbDeleteFile</tabstop>
  <tabstop>ui_leBookLabel</tabstop>
  <tabstop>ui_twSnippets</tabstop>
  <tabstop>ui_pbAddSnippet</tabstop>
  <tabstop>ui_pbDeleteSnippet</tabstop>
  <tabstop>ui_leSnippetLabel</tabstop>
  <tabstop>ui_leSnippetIcon</tabstop>
  <tabstop>ui_tbSnippetIcon</tabstop>
  <tabstop>ui_pteSnippetTex</tabstop>
  <tabstop>ui_spbSnippetPos</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditSnippets</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>622</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>593</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditSnippets</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>622</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>594</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ui_lwSnippetFiles</sender>
   <signal>currentItemChanged(QListWidgetItem*,QListWidgetItem*)</signal>
   <receiver>EditSnippets</receiver>
   <slot>loadSnippetBook(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>123</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>2</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ui_twSnippets</sender>
   <signal>currentCellChanged(int,int,int,int)</signal>
   <receiver>EditSnippets</receiver>
   <slot>loadSnippetItem(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>371</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>478</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ui_leSnippetLabel</sender>
   <signal>textEdited(QString)</signal>
   <receiver>EditSnippets</receiver>
   <slot>updateSnippetLabel(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>588</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>553</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ui_leSnippetIcon</sender>
   <signal>textEdited(QString)</signal>
   <receiver>EditSnippets</receiver>
   <slot>updateSnippetIcon(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>335</y>
    </hint>
    <hint type="destinationlabel">
     <x>358</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ui_pbAddFile</sender>
   <signal>clicked()</signal>
   <receiver>EditSnippets</receiver>
   <slot>addSnippetBook()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>26</x>
     <y>574</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>450</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ui_leBookLabel</sender>
   <signal>textEdited(QString)</signal>
   <receiver>EditSnippets</receiver>
   <slot>updateBookLabel(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>419</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ui_pbAddSnippet</sender>
   <signal>clicked()</signal>
   <receiver>EditSnippets</receiver>
   <slot>addSnippet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>291</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ui_pbDeleteSnippet</sender>
   <signal>clicked()</signal>
   <receiver>EditSnippets</receiver>
   <slot>deleteSnippet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>507</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ui_pbDeleteFile</sender>
   <signal>clicked()</signal>
   <receiver>EditSnippets</receiver>
   <slot>deleteSnippetBook()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>575</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ui_tbSnippetIcon</sender>
   <signal>clicked()</signal>
   <receiver>EditSnippets</receiver>
   <slot>selectIcon()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>569</x>
     <y>335</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>loadSnippetBook(QListWidgetItem*)</slot>
  <slot>loadSnippetItem(int)</slot>
  <slot>updateSnippetLabel(QString)</slot>
  <slot>updateSnippetIcon(QString)</slot>
  <slot>updateSnippetTex()</slot>
  <slot>updateSnippetPos(int)</slot>
  <slot>addSnippetBook()</slot>
  <slot>updateBookLabel(QString)</slot>
  <slot>addSnippet()</slot>
  <slot>deleteSnippet()</slot>
  <slot>deleteSnippetBook()</slot>
  <slot>selectIcon()</slot>
 </slots>
</ui>
